compose/config/validation.py

```diff
@@ -364,19 +364,33 @@ def validate_against_fields_schema(config_file):
 def validate_against_service_schema(config, service_name, version):
     _validate_against_schema(
         config,
-        "service_schema_v{0}.json".format(version),
+        load_jsonschema("service", version),
         format_checker=["ports"],
         service_name=service_name)
 
 
+def load_jsonschema(prefix, version):
+    schema_filename = "{0}_schema_v{1}.json".format(prefix, version)
+    config_source_dir = os.path.dirname(os.path.abspath(__file__))
+
+    if sys.platform == "win32":
+        config_source_dir = config_source_dir.replace('\\', '/')
```

Comment:
Ah yes, I saw that, but I'm not sure why it's there either. This is mostly just a question, it may be necessary. If you can remove it and the appveyor tests pass (they run on windows) then I think it's safe to remove.